src/NuGet.Core/NuGet.LibraryModel/ToolDependency.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.LibraryModel
+{
+    public class ToolDependency
+    {
+        public LibraryRange LibraryRange { get; set; }
```

Comment:
`LibraryRange` includes the ID, version range, and type constraint (which is always set to Package since P2P tools is not supported). I thought this was the clearest model to use, but also considered not using `LibraryRange` at all. Objections? Thoughts?